package.json

```diff
@@ -12,6 +12,7 @@
     "cypress": "^4.6.0",
     "cypress-axe": "^0.8.1",
     "gatsby": "^2.22.0",
+    "gatsby-cli": "^2.12.43",
```

Comment:
This shouldn't be present at all.  Gatsby CLI is only required on the develop machine.

Reply:
If you revert these changes:

- Edits to README.md
- Edits to package.json
- Edits to yarn.lock

I can approve this pull request.